packages/epics/src/spaces/components/parent-breadcrumbs.tsx

```
@@ -0,0 +1,50 @@
+// import { findSpaceById } from '@hypha-platform/core/server';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbSeparator,
+} from '@hypha-platform/ui';
+import { ChevronLeftIcon } from 'lucide-react';
+
+type Breadcrumb = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Breadcrumb'. Consider to delete it or rename it.

'Breadcrumb' is defined here:


(lint/suspicious/noRedeclare)